tests/scripts/task_kg_test.sh

```diff
@@ -34,10 +34,10 @@ conda activate ${DGLBACKEND}-ci
 # test
 if [ "$2" == "cpu" ]; then
     pip3 uninstall dgl
-    pip3 install dgl==0.4.3post2
+    pip3 install dgl
```

Comment:
why CPU version doesn't define the version number, but GPU version does?

Reply:
Thank you for catching this @zheng-da . My mistake - I added the version requirements to the wrong line - it should be correct now